Gaijinizer patch/Scripts.txt

```diff
@@ -3035,40 +3035,39 @@ MOWは、きっと私が根絶やしにしてみせる。
  最後まで、決意を捨てないで……
  =================================================="
 > CONTEXT: Scripts/テキスト出力/172:12
-"Subject: To 'you'
-Sender [Reina]]
+"Sender: Reina
+Subject: To "You"
 
- 《Text》==========================================
-......... Has it arrived properly?
- If you use this machine invented by Dr. Kurosawa.
- I hear that I can reach you and I'm trying.
- I do not know whether it can be transmitted well because of voice input ...
-
-First of all...Thank you. Help me ...
-I'm not going to blame you,
-You just acted in response to his 'determination' ...
-If I sacrifice me in that scene, he will die
- The meaning taken in by Samael is gone ...
-Surely, I thought so.
-
-I do not know when it will be ...
-M.O.W. will surely make me eradicate.
- A forger who afflicted me and him...the heart of this war ...
-It is my guilty to end up with them ...
-So don't worry about 'this world' ...
-
-But what if ...
-If you have the courage to do it again ...
-He also wants me to save him in the world.
-
- I can not but live in this world, even if you
- Having a better end does not matter to me now.
-But if they are 'my girls' in my room ...
-It will surely help you.
-
-What you are doing 'intervention' should have something to do with it.
- Do not throw out the determination until the end ...
- =================================================="#MTLed
+ 《Body》==========================================
+......Has this reached you?
+I heard from Professor Yui that by using this device she invented,
+I can get in touch with you, the "intervener". So I'm giving it a try.
+It uses voice input so I don't know whether this message will be
+transmitted properly...
+
+First of all, thank you for rescuing me. I have no intention of blaming
+you for anything, you simply acted in response to his "determination".
+If I had been sacrificed back then, there would have been no meaning to
+him being taken in by Samael. That's what I thought.
+
+I don't know when it will happen, but at some point M.O.W. will
+definitely try to eradicate me. They're the ringleaders behind
+both his and my suffering... the cause of this war. Settling things
+with them once and for all will be my atonement. So, please don't worry
+about this world.
+
+But if...
+If you have the courage to try again, I'd like you to save him
+in that world.
+
+I have no choice but to continue living in this world, so even if you
+reach a better conclusion in that world, it will have no impact on the
+me here and now. But I'm sure those girls in "My Room" will be able to
+help you.
+
+I believe there's some meaning to your "interventions". Don't throw away
+your determination...
+ =================================================="#Ron
 > END STRING
```

Comment:
This is probably one of the Mercy endings. I'd make a note here to come back to this, I don't think I'm ready to review it yet.